docs/rules.md

```diff
@@ -155,6 +155,9 @@ When a composable function exists solely because it's a `@Preview`, it doesn't n
 
 Related rule: [twitter-compose:preview-public-check](https://github.com/twitter/compose-rules/blob/main/rules/common/src/main/kotlin/com/twitter/compose/rules/ComposePreviewPublic.kt)
 
+NOTE: if you're using Detekt, this may conflict with Detekt's [UnusedPrivateMember rule](https://detekt.dev/docs/rules/style/#unusedprivatemember).
```

Comment:
```suggestion
NOTE: if you are using Detekt, this may conflict with Detekt's [UnusedPrivateMember rule](https://detekt.dev/docs/rules/style/#unusedprivatemember).
```